Classes/Json/JsonEncoder.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
declare strict types line should be below license

Reply:
Added in newest commits